kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -18,7 +18,7 @@ define([
     const t = html.tag,
         div = t('div'),
         select = t('select'),
-        option = t('option');
+        MAX_OPTIONS = 20;
```

Comment:
I wonder if there are likely to be examples of app cells with numerous select elements with more than 20 options and if it's worth checking for that -- i.e. only doing this if the selects are in a bulk cell. Hmmm, on second thoughts, probably not worth it.